Core/Core/Credentials/AccountManager.cs

```diff
@@ -53,8 +54,23 @@ public static async Task<ServerInfo> GetServerInfo(Uri server, CancellationToken
       httpClient
     );
 
+    System.Version version = await gqlClient
+      .GetServerVersion(cancellationToken: cancellationToken)
+      .ConfigureAwait(false);
+    System.Version serverMigrationVersion = new(2, 18, 5);
+
+    string queryString;
+    if (version >= serverMigrationVersion)
+    {
+      queryString = "query { serverInfo { name company migration { movedFrom movedTo } } }";
+    }
+    else
+    {
+      queryString = "query { serverInfo { name company } }";
+    }
+
```

Comment:
This is for backwards compatibility only, we need to discuss with @BovineOx how long should this stay.

Reply:
this is indeed only for backwards compatibility

Reply:
I'm opening a ticket for this and letting Ian know